NEWS.md

```diff
@@ -1,5 +1,126 @@
 MXNet Change Log
 ================
+## 1.2.0
+### New Features - Added Scala Inference APIs
+- Implemented new [Scala Inference APIs](https://cwiki.apache.org/confluence/display/MXNET/MXNetScalaInferenceAPI) which offer an easy-to-use, and Scala Idiomatic and thread-safe high level APIs for performing predictions with deep learning models trained with MXNet (#9678). Implemented new ImageClassifier class provides APIs for classification tasks on a Java BufferedImage using a pre-trained model you provide (#10054). Implemented new ObjectDetector class provides APIs for object and boundary detections on a Java BufferedImage using a pre-trained model you provide (#10229).
+
+### New Features - Added module to import ONNX models into MXNet
+- Implemented new ONNX module in MXNet offers an easy to use API to import ONNX models into MXNet's symbolic interface (#9963). Checkout the example on how you could use this API to import ONNX models and perform inference on MXNet. 
```

Comment:
And in "Checkout the example ..." part. The example should link to one of these two - https://github.com/apache/incubator-mxnet/blob/master/example/onnx/super_resolution.py or https://github.com/apache/incubator-mxnet/blob/master/docs/tutorials/onnx/super_resolution.md